packages/core/src/client/index.ts

```diff
@@ -585,6 +583,24 @@ export class Web3InboxClient {
         }`
       );
     }
+  }, this.MARK_NOTIFICATIONS_AS_READ_DEBOUNCE_TIME)
+
+  /**
+   * Mark specified notifications as read for a account & domain subscription
+   * @param {string[]} notificationIds - Account to get subscription message history for, defaulted to current account
+   * @param {string} [account] - Account to get subscription message history for, defaulted to current account
+   * @param {string} [domain] - Domain to get subscription message history for, defaulted to one set in init.
+   */
+  public markNotificationsAsRead(
+    notificationIds: string[],
+    account?: string,
+    domain?: string
+  ) {
+    for(const notificationId of notificationIds) {
+      Web3InboxClient.notificationsToRead.add(notificationId)
```

Comment:
The processing of `notificationsToRead` doesn't respect the max size of this array

Reply:
What do you mean